driver-core/src/main/com/mongodb/internal/connection/UnixSocketAddressHelper.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.internal.connection;
+
+import jnr.unixsocket.UnixSocketAddress;
+
+public final class UnixSocketAddressHelper {
```

Comment:
I gather this class is needed as an indirection to avoid ClassNotFoundException when the optional dependency is not available but not actually needed?  If so, a short comment to explain would be useful.

Reply:
With the `UnixServerAddress` refactor this is no longer required - deleted.